app/src/main/java/com/eveningoutpost/dexdrip/FakeNumbers.java

```diff
@@ -44,5 +46,24 @@ public void onClick(View v) {
             }
 
         });
+        
+        button = (Button)findViewById(R.id.StartTest);
+        button.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+                ActiveBgAlert aba = ActiveBgAlert.getOnly();
+                ActiveBgAlert.ClearData();
+                ActiveBgAlert.Create("some string", true, new Date().getTime());
```

Comment:
This seems very strange. 
We have the line new Date().getTime() about 50 times in our code.
How are you trying to build?
Is the problem with "import java.util.Date;" ?